GOVERNANCE.md

```diff
@@ -122,3 +122,11 @@ The Advisory Committee provides perspective and advice to the Technical Steering
 * A decision may be revisited if new technical information becomes available.
 
 * The Advisory Committee, TSC and Working Groups may hold calls, video conferences, and face to face meetings. These meetings shall be announced sufficiently in advance with a published agenda. A mechanism by which the community can propose items for the agenda shall be provided.
+
+## Contributor License Agreement
+
+* The AMP Project requires all Owners, Collaborators, and Contributors who open a pull request, to either sign an individual Contributor License Agreement (CLA) or be covered by a corporate CLA in order to protect contributors and users in issues of intellectual property.
```

Comment:
> pull request, to either

I don't believe a comma here is necessary.

Reply:
Indeed! Fixed.